src/main/java/seedu/address/model/project/exceptions/ProjectNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.project.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified project.
+ */
+public class ProjectNotFoundException extends RuntimeException {}
```

Comment:
Nothing implemented here

Reply:
This was copied over from person class. Has same functionality as RuntimeException.